docs/components/icons/vue.md

```diff
@@ -0,0 +1,11 @@
+## Import
+
+```js
+import { IconBag16 } from '@warp-ds/icons/vue';
+```
+
+### Usage
+
+```html
+  <icon-bag-16 class="s-icon" />
```

Comment:
Don't know if it makes any difference in Vue, but in the [README](https://github.com/warp-ds/icons#vue) in @warp-ds/icons it's documented that Vue icons should be used like `<icon-bag16>`. Would be good to align there, so just let me know if I should update the readme 😊

Reply:
https://github.com/warp-ds/tech-docs/pull/102/commits/43743edb7715c6c5552fbac024c45f2b1bce8511